src/v/storage/segment_deduplication_utils.cc

```diff
@@ -116,6 +116,25 @@ ss::future<model::offset> build_offset_map(
             cfg.asrc->check();
         }
         auto seg = *iter;
+        if (seg->index().has_clean_compact_timestamp()) {
```

Comment:
nit: just making sure, is there anywhere in code comments that describes what it means to be clean compacted?

Reply:
[1](https://github.com/redpanda-data/redpanda/blob/4045b9f90f9be8a814c3bb15941f5b15bb690b64/src/v/storage/disk_log_impl.cc#L691-L692), [2](https://github.com/redpanda-data/redpanda/blob/4045b9f90f9be8a814c3bb15941f5b15bb690b64/src/v/storage/disk_log_impl.h#L189-L197)